device/device.yaml

```diff
@@ -50,4 +50,9 @@ components:
             The properties of BGP router and its children, 
             such as BGPv4, BGPv6 peers and their route ranges.
           $ref: './bgp/bgp.yaml#/components/schemas/Device.BgpRouter'
+
+        rsvp:
+          description: >-
+            The properties of RSVP Neighbor and its children.
+          $ref: './rsvp/rsvp.yaml#/components/schemas/Device.Rsvp'
```

Comment:
If we should rename "Device.Rsvp" to "Device.RsvpNeighbor" or something explains "Rsvp" construct?

Reply:
Rsvp -> contains configuration for rsvp neighbor,head and tail end lsps so have kept Device.Rsvp